retriever/engines/mysql.py

```diff
@@ -39,6 +41,21 @@ class engine(Engine):
                       "{db}.{table}"),
                      ]
 
+    def check_credentials(self):
+        self.get_input()
+        self.sql_lookup = {"user": self.opts["user"], "password": self.opts["password"], "host": self.opts["host"], "port": self.opts["port"]}
```

Comment:
@henrykironde I also thought of just modifying the `self.opts` dictionary but I thought making a separate dictionary would make the code more clean and readable. I will change the functioning to change the `sel.opts` dictionary itself if you feel so. Kindly let me know :smile: